.github/workflows/py-build.yml

```diff
@@ -57,6 +72,15 @@ jobs:
         run: python -m build
         working-directory: tools/cell_census_builder/
 
-      - name: Build image
-        run: docker build --build-arg=COMMIT_SHA=$(git rev-parse --short HEAD) -t cell-census-builder .
+      - name: Build, tag, and push docker image to Amazon ECR
+        env:
+          REGISTRY: ${{ secrets.ECR_REGISTRY }}
+          REPOSITORY: cell-census-builder
+          IMAGE_TAG: ${{ github.sha }}
+        run: |
+          docker build --build-arg=COMMIT_SHA=$(git rev-parse --short HEAD) -t $REGISTRY/$REPOSITORY:$IMAGE_TAG .
```

Comment:
lets fix the COMMIT_SHA to match the Makefile (use `git describe`)

Reply:
Actually, we can get it using the github variable directly (see `IMAGE_TAG` above)

Reply:
Should be `github.ref_name`. Let's try and see what happens.